packages/timeline-state-resolver/src/integrations/atem/stateBuilder.ts

```diff
@@ -123,8 +116,20 @@ export class AtemStateBuilder {
 	private _applyMixEffect(mapping: MappingAtemMixEffect, content: TimelineContentAtemME): void {
 		if (typeof mapping.index !== 'number' || mapping.index < 0) return
 
-		const stateMixEffect = AtemStateUtil.getMixEffect(this.#deviceState, mapping.index)
-		deepExtend(stateMixEffect, _.omit(content.me, 'upstreamKeyers'))
+		const stateMixEffect = deepMerge(
+			AtemStateUtil.getMixEffect(this.#deviceState, mapping.index),
+			_.omit(content.me, 'upstreamKeyers', 'transitionPosition')
+		)
+		this.#deviceState.video.mixEffects[mapping.index] = stateMixEffect
+		if (typeof content.me.transitionPosition === 'number') {
```

Comment:
Minor readability note:
To me, `typeof content.me.transitionPosition === 'number'` implies that there are other possible types, a `content.me.transitionPosition !== undefined` is slightly less suprising in this case, I think.